runtime/layers/fpm/bootstrap

```diff
@@ -32,7 +32,12 @@ if (! is_file($handlerFile)) {
     $lambdaRuntime->failInitialization("Handler `$handlerFile` doesn't exist");
 }
 
+$timeout = getenv('BREF_FPM_TIMEOUT');
+
 $phpFpm = new FpmHandler($handlerFile);
+
+$phpFpm->setTimeout((int) $timeout);
```

Comment:
Shouldn't this be called only when a timeout env Var is provided?

Because the variable is set by the serverless framework plugin, it may not be set when deploying with sam/cdk/cloudformation